content/aws_marketplace.md

```diff
@@ -42,83 +42,157 @@ AWS provides 5 VPCs for each region. If you require more VPCs, please contact [A
 
 1. Enter your **Username** and **Password** and select **Sign In**.
-![ ](/images/chef_automate_login.png "Chef Automate Login")
+
+    ![ ](/images/chef_automate_login.png "Chef Automate Login")
 
 1. Fill out the registration form and [Accept the Chef License](https://docs.chef.io/chef_license_accept/).
 
 1. Select **Register** to enter Chef Automate.
-![ ](/images/WelcomePage.png "Welcome Page")
+    
+    ![ ](/images/WelcomePage.png "Welcome Page")
 
 1. Congratulations! You've started Chef Automate!
-![ ](/images/DashboardsPage.png "Dashboards Page")
+
+    ![ ](/images/DashboardsPage.png "Dashboards Page")
 
 ## Add Chef Servers
 
-1. Add Chef-Server Details, select the Add Chef Infra Server Button.
-![ ](/images/chef_automate_add_server.png "Add Chef Server")
+1. Add Chef-Server Details, and select the Add Chef Infra Server Button.
+
+    ![ ](/images/chef_automate_add_server.png "Add Chef Server")
 
 1. Enter the server name, FQDN, and IP address. Then select **Add Chef Infra Server** to create the server.
 
-    - Name: Add Proper Name for the Sever.
+    - Name: Add the Proper Name for the Sever.
     - FQDN: It would be the same as Automate FQDN.
     - IP Address: Public IP Address of the EC2-Instance.
 
     ![Add Chef Infra Server Form](/images/automate/add-chef-server-popup-menu.png)
 
 1. The Chef Infra Server will appear in the list of servers. Select the server and view information about it.
-![Select a server from the list](/images/chef_automate_single_server.png "Single Server View")
+
+    ![Select a server from the list](/images/chef_automate_single_server.png "Single Server View")
 
 1. Select **Add Chef Organization**.
-{{< figure src="/images/chef_automate_add_org_page.png" style="width: 30%;" >}}
+
+    {{< figure src="/images/chef_automate_add_org_page.png" style="width: 30%;" >}}
 
 1. Enter the following information:
 
     - Name: **demo**
     - Admin User: **admin**
-    - Admin Key: _copy the key from starter kit_
+    - Admin Key: _copy the key from the starter kit_
 
 1. Select **Add Chef Organization**.
-![Select the Add Chef Organization button to complete this action](/images/OrgPageDetails.png)
+    
+    ![Select the Add Chef Organization button to complete this action](/images/OrgPageDetails.png)
+
+## AWS EC2 Access
+
+The AWS EC2 instance connects functionality can be used to access the EC2 machine. Using the same instead of SSH to the EC2 device is recommended.
+
+1. Select the EC2 machine and select the connect option as shown below:
+
+    ![Add Chef Infra Server Form](/images/select-connect-option-in-ec2machine.png)
+
+1. Select the **Connect** button on the new screen shown below:
+
+    ![Add Chef Infra Server Form](/images/select-button-on-ec2machine.png)
+
+The above action will launch a new tab for EC2 connect page, as shown below:
+
+    ![Add Chef Infra Server Form](/images/new-tab-of-ec2-machine.png)
+
+Now, you are in the EC2 machine, where you can run the necessary commands.
```

Comment:
This isn't quite correct. The user isn't _in_ anything, although they probably have their EC2 instance running. Or something like that. Also, too many commas.